hibike/hibike_tester.py

```diff
@@ -1,4 +1,5 @@
 import hibike_process
+import time
```

Comment:
we did to test specific functionality.

That functionality has been reverted for the commit.  But given the number of times that we've had to put time.sleep() in the while loop, and given how important time.sleep() is to having loops run at real rates, I suggest we leave this import in.

Possibly even have a time.sleep() in the while true loop to demonstrate its use